apps/bfd-pipeline/bfd-pipeline-ccw-rif/src/main/java/gov/cms/bfd/pipeline/ccw/rif/CcwRifLoadJob.java

```diff
@@ -356,78 +365,44 @@ && dataSetIsAvailable(secondManifestToProcess.get())) {
        */
       rifFiles.forEach(S3RifFile::cleanupTempFile);
     } else {
+      // TODO BEGIN remove once S3 file moves are no longer necessary.
       /*
        * If here, Synthea pre-validation has failed; we want to move the S3 incoming
        * files to a failed folder; so instead of moving files to a done folder we'll just
        * replace the manifest's notion of its Done folder to a Failed folder.
        */
       manifestToProcess.setManifestKeyDoneLocation(S3_PREFIX_FAILED_SYNTHETIC_DATA_SETS);
+      // TODO END remove once S3 file moves are no longer necessary.
+
+      /*
+       * If here, Synthea pre-validation has failed; we want to mark the data set as rejected int he database.
+       */
+      dataSetQueue.markAsRejected(manifestRecord);
```

Comment:
A bit of tech debt, but the synthea pre-validation doesnt even run now that we are permanently in idempotent mode. Eventually we should probably set up a new checkbox in the automation which passes a flag to run or not run this validation (and replace the idempotent check on line 300 with that flag), so ideally this will still be valuable one day.

No action to take here, just reminded me of that.